src/V4/PeanutV4.sol

```diff
@@ -38,6 +38,7 @@ import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
 import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
 import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import {IL2ECO} from "../util/IL2ECO.sol";
+import {IEIP3009} from "../util/IEIP3009.sol";
```

Comment:
Can't we import this from an established library like OZ? Although being just an interface it's fine

Reply:
I haven't found anything. 3009 is still in the draft status, that is probably the reason.

https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2436